### DISCUSSION TITLE
Get Uploading Status (Yes/No) of file getting uploaded in one drive

### DISCUSSION THREAD

**Discussion by TejasPatadiya10** [microsoftgraph/msgraph-sdk-dotnet#2477] [Q&A]
We have kept large file for uploading and on the other end we are retrieving all the files from that one drive.

We want to fetch only items which are fully uploaded but currently we are getting partially uploaded file as well using graph api.

how can we get only items which are fully uplaoded or some how we can ignore them. please suggest

**Comment by andrueastman** [comment]
Thanks for raising this @TejasPatadiya10 

As you provide a `fileName` when creating the upload, is possible to use the filename or the path to create an exclusion list? I believe, on successful completion of the upload, the API should return [the success object](https://learn.microsoft.com/en-us/graph/api/driveitem-createuploadsession?view=graph-rest-1.0&preserve-view=true#response-3) that can be used to remove the item from the list.
